frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -864,6 +869,19 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
         )}
       </SectionBox>
 
+      {ephemeralContainers.length > 0 && (
+        <SectionBox title={t('translation|Ephemeral Containers')}>
+          {ephemeralContainers.map((container: any) => (
```

Comment:
The type should be KubeContainer. Like in the init containers case.

Reply:
done!